resources/tdm_gcc.rb

```diff
@@ -87,6 +87,19 @@
       overwrite true
     end
   end
+
+  # Patch time.h headers for compatibility with winpthreads.
+  # These patches were made for binutils 2.25.1 for 32-bit TDM GCC only.
```

Comment:
D'oh - I'm stupid - this was that other fix....  I was looking at the wrong repo.  This one I do have comments and explanations for.  Yes these are things that should be fixed upstream but they are never going to be because the majority of the relevant libraries (and there are 3 or 4 independent player involved) were abandoned years ago.  I'll document what my patch does.